src/Components/IndexScene/InterceptBanner.tsx

```diff
@@ -0,0 +1,56 @@
+import { Alert } from '@grafana/ui';
+import React from 'react';
+
+export function InterceptBanner(props: { interceptDismissed: boolean; onRemove: () => void }) {
+  return (
+    <>
+      {!props.interceptDismissed && (
+        <Alert
+          severity={'info'}
+          title={'Explore logs is under active development in a preview state.'}
```

Comment:
maybe we can say,

"Welcome to Explore Logs (Preview) - we're in active development so please expect things to change.

Check out our video or dive into our Get Started docs for tips on how to use the app.

Let us know what we got right and what we got wrong by sending us your feedback."

Can we put a thumbnail of the video in there (it's fine if it pops up in a new window)?

Reply:
there will be a Get Started doc page: https://github.com/grafana/explore-logs/pull/456/files#diff-885722bddf5b70461db4e3b67474d96e8e0e3aafa781f7c3005cc7a38fa91f63